docs/querying/sql-jdbc.md

```diff
@@ -82,3 +90,112 @@ statement.setString(1, "abc");
+
+### Get the metadata for a datasource
+
+Metadata, such as column names, is available either through the [`INFORMATION_SCHEMA`](../querying/sql-metadata-tables.md) table or through `connect.getMetaData()`. The following example uses the `INFORMATION_SCHEMA` table to retrieve and print the list of column names for the `wikipedia` datasource that you loaded during a previous tutorial.
+
+```java
+import java.sql.*;
+import java.util.Properties;
+
+public class JdbcListColumns {
+
+    public static void main(String args[]) throws SQLException
+    {
+        // Connect to /druid/v2/sql/avatica/ on your Router. 
+        // You can connect to a Broker but must configure connection stickiness if you do. 
+        String url = "jdbc:avatica:remote:url=http://localhost:8888/druid/v2/sql/avatica/";
+
+        String query = "SELECT COLUMN_NAME,* FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'wikipedia' and TABLE_SCHEMA='druid'";
+        // Set any connection context parameters you need here
+        // Or leave empty for default behavior.
+        Properties connectionProperties = new Properties();
+
+        try (Connection connection = DriverManager.getConnection(url, connectionProperties)) {
+            try (
+                    final Statement statement = connection.createStatement();
+                    final ResultSet rs = statement.executeQuery(query)
+            ) {
+                while (rs.next()) {
+                    String columnName = rs.getString("COLUMN_NAME");
+                    System.out.println(columnName);
+                }
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+}
+```
+
+### Query data
+
+Now that you know what columns are available, you can start querying the data. The following example queries the datasource named `wikipedia` for the timestamps and comments from Japan. It also sets the [query context parameter](../querying/sql-query-context.md) `sqlTimeZone` . Optionally,  you can also parameterize queries by using [dynamic parameters](#dynamic-parameters).
+
+```java
+import java.sql.*;
+import java.util.Properties;
+
+public class JdbcCountryAndTime {
+
+    public static void main(String args[]) throws SQLException
+    {
+        // Connect to /druid/v2/sql/avatica/ on your Router. 
+        // You can connect to a Broker but must configure connection stickiness if you do. 
+        String url = "jdbc:avatica:remote:url=http://localhost:8888/druid/v2/sql/avatica/";
+
+        //The query you want to run.
+        String query = "SELECT __time, isRobot, countryName, comment FROM wikipedia WHERE countryName='Japan'";
+        // Set any connection context parameters you need here
+        // Or leave empty for default behavior.
+        Properties connectionProperties = new Properties();
+        connectionProperties.setProperty("sqlTimeZone", "America/Los_Angeles");
```

Comment:
I think we can add a comment here that this is optional. Users will get back UTC is this is not set (which is recommended)

It's usually preferred to do timezone adjustments on the client side. The browser knows the timezone that the user is in.